client/app/dbaas/dbaas-metrics/token/delete/metrics-token-delete.html

```diff
@@ -1,16 +1,10 @@
-<cui-modal>
-    <cui-modal-header data-on-dismiss="$ctrl.cancel()"></cui-modal-header>
-    <cui-modal-body data-title=":: 'metrics_token_delete_title' | translate">
-        <oui-spinner data-ng-show="$ctrl.loading"></oui-spinner>
-        <p data-translate="metrics_token_delete_message"></p>
-    </cui-modal-body>
-    <cui-modal-footer>
-        <cui-cancel-button
-          data-text="'metrics_token_cancel' | translate"
-          data-ng-click="$ctrl.cancel()"></cui-cancel-button>
-        <cui-submit-button
-          data-text="'metrics_token_delete' | translate"
-          data-ng-click="$ctrl.confirm()"
-          data-ng-disabled="$ctrl.loading"></cui-submit-button>
-    </cui-modal-footer>
-</cui-modal>
+<oui-modal
+    data-title="{{ ::'metrics_token_delete_title' | translate }}"
+    data-secondary-action="$ctrl.cancel()"
+    data-secondary-label="{{ ::'metrics_token_cancel' | translate }}"
+    data-primary-action="$ctrl.confirm()"
+    data-primary-label="{{ ::'metrics_token_delete' | translate}}"
+    data-on-dismiss="$ctrl.cancel()"
+    data-loading="$ctrl.loading">    
+    <p class="oui-modal__text" data-translate="metrics_token_delete_message"></p>
```

Comment:
I'm a little curious about why you are using a CSS class `` on your `<p>` element.

According to the documentation, here is a snippet:

```html
<oui-modal heading="Modal title" primary-action="$ctrl.confirm = true" primary-label="Ok" secondary-action="$ctrl.cancel = true" secondary-label="Cancel" on-dismiss="$ctrl.cancel = true">
  Modal content
</oui-modal>
```

No CSS class needed on children elements.

Reply:
@varun257 @Netnix I'm still curious about it! Hope that I'll have an answer since the merge!